db-pull-on-boot/tasks/main.yml

```diff
@@ -0,0 +1,21 @@
+---
+- name: Copy over the db-pull.sh file
+    template:
+      src: db-pull.sh.j2
+      dest: {{ chia_root }}/db-pull.sh
+
+- name: Install the DB unit file
+  become: yes
```

Comment:
```suggestion
```

We can probably just call become when we include the role (otherwise, the systemd task below will also need become)